Tests/RxSwiftTests/Observable+ZipTests+arity.tt

```diff
@@ -33,7 +33,7 @@ extension ObservableZipTest {
 
```

Comment:
This is the source of truth for `Tests/RxSwiftTests/Observable+ZipTests+arity.swift`. This should probably be fixed also.